runtime/wasm/src/module/mod.rs

```diff
@@ -1170,6 +1170,15 @@ impl<C: Blockchain> WasmInstanceContext<C> {
         let entity_type: String = asc_get(self, entity_type_ptr, gas)?;
         let id: String = asc_get(self, id_ptr, gas)?;
         let field: String = asc_get(self, field_ptr, gas)?;
+
+        error!(
+            &self.ctx.logger,
+            "store_load_related";
+            "type" => &entity_type,
+            "id" => &id,
+            "field" => &field,
+        );
+
```

Comment:
This log looks unrelated.

Reply:
oops, put it there while i was debugging another issue, removed it. TY